packages/web/components/cards/app-display-card.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { FunctionComponent, memo } from "react";
+
+import { EventName } from "~/config";
+import { useAmplitudeAnalytics } from "~/hooks";
+
+import { Icon } from "../assets";
+import { IconLink } from "./icon-link";
+
+export const AppDisplayCard: FunctionComponent<{
+  title?: string;
+  subtitle?: string;
+  imageUrl?: string;
+  twitterUrl?: string;
+  githubUrl?: string;
+  externalUrl?: string;
+  mediumUrl?: string;
+  index: number;
+}> = memo(
+  ({
+    title,
+    subtitle,
+    imageUrl,
+    twitterUrl,
+    githubUrl,
+    externalUrl,
+    mediumUrl,
+    index,
+  }) => {
+    const { logEvent } = useAmplitudeAnalytics();
+    const handleAppClicked = () => {
+      logEvent([
+        EventName.AppStore.appClicked,
+        { appName: title, isFeatured: false, isBanner: false, position: index },
+      ]);
+    };
+    return (
+      <>
+        <style jsx>{`
+          .app-display-card:hover .card-image {
+            transform: scale(1.15);
+          }
+          .card-image {
+            background-image: url(${imageUrl});
+            transition: transform 0.3s ease-in-out;
+          }
+        `}</style>
```

Comment:
This could also be added with tailwind arbitrary values. I'm not a fan of mixing styling solutions. It adds more complexity

Reply:
Tailwind doesn't appear to be able to support this use case. I couldn't get the animations to work correctly in its engine.